GOT-StudentVersion/GOT-StudentVersion/Controllers/GOTETableViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  GOTETableViewController.swift
+//  GOTE Project
+//
+//  Created by Mariel Hoepelman on 8/18/19.
+//  Copyright © 2019 Mariel Hoepelman. All rights reserved.
+//
+
+import UIKit
+
+class GOTETableViewController: UITableViewController {
+    var episode = GOTEpisode.allEpisodes
+    
+    @IBOutlet weak var searchBarOulet: UISearchBar!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+//        searchBarOulet.delegate = self
+    }
+
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 7
```

Comment:
Avoid hard coding these values. This can break your code if your model changes. 

 return arrOfSeasons.count